frontend/public/components/build.tsx

```diff
@@ -38,6 +39,11 @@ export const BuildsDetails: React.SFC<BuildsDetailsProps> = ({obj: build}) => {
   const duration = formatBuildDuration(build);
 
   return <div className="co-m-pane__body">
+    <div className="row">
```

Comment:
Maybe it's OK, but this is going to add some empty DOM nodes if it's not actually a build pipeline

Reply:
Maybe just check `build.spec.strategy.type === 'JenkinsPipeline'`?

Reply:
Yeah, def plenty reasonable. Lets go that route.